### ISSUE TITLE
Say just "stream" instead of "payment stream" in NFT descriptor

### ISSUE THREAD

**Issue by PaulRBerg** [sablier-labs/flow#357]
To keep Flow in sync with the terminology in Lockup:

https://github.com/sablier-labs/flow/blob/356d652382ca37acc46397155b3b98a62e61ad40/src/FlowNFTDescriptor.sol#L27

**Comment by andreivladbrg** [comment]
@PaulRBerg I have a question regarding this change.

You consider it very important? Because otherwise, since there is no other change to the `NFTDescriptor` contract we can keep the same without deployment them again. This is a mistake I realized during the v1.0.0 deployments.

**Comment by PaulRBerg** [comment]
Fair enough. We can keep it as is to avoid a deployment of the NFT descriptor!
